platform/packages/platform/src/coin_legacy.rs

```diff
@@ -185,15 +188,15 @@ mod test {
     #[test]
     fn from_cosmwasm() {
         let c1 = super::from_cosmwasm::<SuperGroupTestC2>(CosmWasmCoin::new(
-            12,
+            12u32,
```

Comment:
[Minor] Choosing `u32` is a weird middle ground to me as it's not the minimal type that can hold the value, `u8`, nor is it a value which won't require any extending, `u128`.